packages/ui/src/2_molecules/Header/Header.module.css

```diff
@@ -0,0 +1,19 @@
+.header {
+  @apply bg-gray-90 h-16 w-full flex align-middle justify-between px-11;
+
+  & > div {
+    @apply flex items-center;
+  }
+}
+
+.logo {
```

Comment:
This isn't used anywhere but I guess it should be because logo has some padding in the design (and if not, apply the suggestions in this file so we have the desired spacing).

Reply:
It just need to be deleted, padding added in the header and in menuItems